src/js/menu/menu-button.js

```diff
@@ -29,6 +29,14 @@ class MenuButton extends ClickableComponent {
 
     this.update();
 
+    /**
+     * Hide the menu if the number of items is less than or equal to this threshold. This defaults
+     * to 0 and whenever we add items to the menu we'll increment it.
+     *
+     * @type {Number}
```

Comment:
Correction @gkatsev : It should be `@protected` because descendants should be allowed access.